code/main/edge/edgeRequestWorker.brs

```diff
@@ -132,16 +133,58 @@ function _adb_EdgeRequestWorker() as object
             return responseArray
         end function,
 
-        _processRequest: function(eventData as object, ecid as string, configId as string, requestId as string, path as string, edgeDomain = invalid as dynamic) as object
-            requestBody = m._createEdgeRequestBody(eventData, ecid)
+        _processRequest: function(eventData as object, ecid as string, datastreamId as string, requestId as string, path as string, meta as object, edgeDomain as dynamic) as object
+            ''' Append config overrides to meta if set in eventData.config
+            meta = m._appendConfigOverridesToMeta(meta, eventData.config, datastreamId)
 
-            url = _adb_buildEdgeRequestURL(configId, requestId, path, edgeDomain)
+            ''' Override the datastreamId if set in eventData.config
```

Comment:
Can you reword this comment to reflect what the function is doing?

Reply:
Can you also update this function comment?
https://github.com/adobe/aepsdk-roku/pull/152/files#diff-b7b00e7cee0f49d4f627be8f64df686c4e8410614c93830baac78cf755b18c5bL28